Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -11937,8 +11943,13 @@ def pan_os_get_running_config(args: dict):
     if args.get("target"):
         params["target"] = args.get("target")
 
+    if str(args.get("filename")) != 'running_config':
+        file_name = str(args.get("target")) + '_' + str(args.get("filename")) + '_running_config'
+    else:
+        file_name = str(args.get("filename"))
+
```

Comment:
```suggestion
    file_name_arg = args.get("filename")
    target = args.get("target")
    if file_name_arg != 'running_config':
        file_name =  target + '_' + file_name_arg + '_running_config'
    else:
        file_name = file_name_arg

```
The args will be in str type when getting them from the args